runtime/compiler/p/codegen/J9PPCWatchedStaticFieldSnippet.hpp

```diff
@@ -0,0 +1,67 @@
+/*******************************************************************************
+ * Copyright (c) 2000, 2019 IBM Corp. and others
```

Comment:
For new files it should be 
```
 * Copyright (c) 2019, 2019 IBM Corp. and others
```

Similarly in `J9PPCWatchedInstanceFieldSnippet.hpp` at least.